gpkit/constraints/relax.py

```diff
@@ -28,8 +28,6 @@ class ConstraintsRelaxedEqually(ConstraintSet):
     """
 
     def __init__(self, constraints):
-        if not isinstance(constraints, ConstraintSet):
-            constraints = ConstraintSet(constraints)
```

Comment:
I think the docstrings throughout this module be updated to say `constraints : ConstraintSet` instead of `constraints : iterable`? (any old iterable won't have `as_posyslt1`, which is called below.